modules/recommended-etcd-practices.adoc

```diff
@@ -64,8 +57,12 @@ $ sudo docker run --volume /var/lib/etcd:/var/lib/etcd:Z quay.io/openshift-scale
 ----
 --
 
-The output reports whether the disk is fast enough to host etcd by comparing the 99th percentile of the fsync metric captured from the run to see if it is less than 10 ms.
+The output reports whether the disk is fast enough to host etcd by comparing the 99th percentile of the fsync metric captured from the run to see if it is less than 20 ms. These are some of the most important etcd metrics that could be potentially affected by I/O performance:
```

Comment:
Suggested edit to the second sentence: `A few of the most important etcd metrics that might be affected by I/O performance are as follows:`